Bot/TynibotHost.cs

```diff
@@ -53,7 +55,7 @@ public TynibotHost()
 
         public async Task RunAsync(
             BotSettings settings,
-            Func<LogMessage, Task> logFunction,
+            Func<string, Task> logFunction,
```

Comment:
Changed the logging passthrough contract so that other libraries besides Discord could use a generic string logger.